code/modules/clothing/suits/ego_gear/realized.dm

```diff
@@ -374,7 +374,9 @@
 	if(!CanSpawn)
 		return
 	addtimer(CALLBACK(src, .proc/Reset,user), 10 SECONDS)
-	new/mob/living/simple_animal/hostile/naked_nest_serpent_friend(get_turf(user))
+	playsound(get_turf(user), 'sound/misc/moist_impact.ogg', 30, 1)
+	var/mob/living/simple_animal/hostile/naked_nest_serpent_friend/W = new/mob/living/simple_animal/hostile/naked_nest_serpent_friend(get_turf(user))
```

Comment:
```suggestion
	var/mob/living/simple_animal/hostile/naked_nest_serpent_friend/W = new(get_turf(user))
```
You don't have to type `new /path/to/object` when the variable you define is already that exact path-type.